src/api-docs/source/includes/_node_rpc_names.md

```diff
@@ -486,6 +486,75 @@ Name | Default |  Description
 --------- | --------- | --------- | -----------
 claim | Required | raw serialized hex-string
 
+## getdnssecproof
+
+```javascript
+let name;
+```
+
+```shell--vars
+name='foo.bar';
+```
+
+```shell--cli
+hsd-rpc getdnssecproof "$name"
+```
+
+```shell--curl
+curl $url \
+    -X POST \
+    --data "{\"method\":\"getdnssecproof\",\"params\":["\"$name\""]}"
```

Comment:
The other curl examples in the docs avoid escaping double quotes by wrapping the entire json in single quotes, example:

```
curl $url \
  -X POST \
  --data '{
    "method": "getblock",
    "params": [ "'$blockhash'", '$verbose', '$details' ]
  }'
```